### ISSUE TITLE
.sol support?

### ISSUE THREAD

**Issue by JoaoieP** [torokati44/ruffle-android#56]
I am mostly interested in playing the mud and blood series. Mud and Blood 2 appears to work fine aside from the ability so save progress. I believe this is related to .sol files which afaik are now supported by ruffle. Is this on the planned additions and do you think it's possible for a dev with no rust or android experience to pick this up easily?

**Comment by torokati44** [comment]
Yeah, it would certainly be nice to support saving, it's just that I don't really have time to dedicate to this project lately, sorry...
